modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -598,6 +598,55 @@ definition {
 		return ${objectEntryId};
 	}
 
+	macro addObjectValidation {
+		Variables.assertDefined(parameterList = "${objectName},${labelName},${script},${validationType}");
+
+		var portalURL = JSONCompany.getDefaultPortalURL();
+		var userLoginInfo = JSONUtil2.formatJSONSuperAdminUser();
+		var objectId = JSONObject.getObjectId(
+			objectName = ${objectName},
+			userEmailAddress = ${userEmailAddress},
+			userPassword = ${userPassword});
+
+		if (!(isSet(userEmailAddress))) {
+			var userEmailAddress = "test@liferay.com";
+		}
+
+		if (!(isSet(userPassword))) {
+			var userPassword = "test";
+		}
+
+		if (!(isSet(errorMessage))) {
+			var errorMessage = "Please enter a valid entry.";
+		}
+
+		var single_quote = "'";
+		var scape_quotes = "'\''";
```

Comment:
@davisaints did you mean escape_quotes? sorry I missed this in the first review!